test_runner/build.gradle.kts

```diff
@@ -11,6 +14,53 @@ plugins {
     kotlin("jvm") version Versions.KOTLIN
 
     id("io.gitlab.arturbosch.detekt") version Versions.DETEKT
+    id("com.jfrog.bintray") version "1.8.4"
+    id("maven-publish")
+}
+
+// https://bintray.com/flank/maven
+// https://github.com/bintray/gradle-bintray-plugin#readme
+bintray {
+    user = System.getenv("MVN_USER")
+    key = System.getenv("MVN_KEY")
+    publish = true
+    setPublications("mavenJava")
+    pkg(closureOf<BintrayExtension.PackageConfig> {
+        repo = "maven"
+        name = "flank"
+        userOrg = "flank"
+        setLicenses("Apache-2.0")
+        vcsUrl = "https://github.com/TestArmada/flank.git"
+        version(closureOf<BintrayExtension.VersionConfig>{
+            name = System.getenv("MVN_VERSION")
+            vcsTag = System.getenv("MVN_REVISION")
+            released = Date().toString()
+
+        })
+    })
+}
+
+publishing {
+    publications {
+        register("mavenJava", MavenPublication::class) {
+            from(components["java"])
+
+            groupId = "flank"
```

Comment:
should the groupId include `testarmada` or some other unique identifier?

Reply:
I think flank is unique? I hesitate to use `testarmada` since I'm not sure that'll be true forever.

Reply:
Makes sense. Flank is an English word, I think its not the best to use english words as group ids. Usually it is a website identifier.

Anyways, the maven jar is working now! I really appreciate it!